device/pools.go

```diff
@@ -7,14 +7,13 @@ package device
 
 import (
 	"sync"
-	"sync/atomic"
 )
 
 type WaitPool struct {
 	pool  sync.Pool
 	cond  sync.Cond
 	lock  sync.Mutex
-	count atomic.Uint32
+	count uint32
```

Comment:
maybe comments while we're here?

```go
    count uint32 // Gets calls not yet Put back
```

or something?

Reply:
updated https://github.com/tailscale/wireguard-go/pull/28/commits/87d0a6ae42be336d1fee5f213d293ba17f407d0e